### ISSUE TITLE
Unicode chars are broken in stacktrace

### ISSUE THREAD

**Issue by throwable-one** [behave/behave#453]
```
def foo(stop):
        raise Exception(u"по русски")
```

Result:
```
       File "features/steps/steps.py", line 8, in foo
          raise Exception(u"Ð¿Ð¾ Ñ�Ñ�Ñ�Ñ�ÐºÐ¸") <-- This is not
      Exception: по русски <-- This is OK
```

It happens here (https://github.com/behave/behave/blob/master/behave/model.py#L1299) becase ``traceback.format_exc()`` creates incorrect text. You then convert it using ``_text()`` and result is also bad.

To fix it, you may take ``e.message`` which is correct and ``traceback.format_tb(sys.exc_info()[2])`` which is also correct.

**Comment by joshsleeper** [comment]
I'm seeing this same problem in 1.2.5 right now.

**Comment by jenisys** [comment]
FIXED IN: behave-1.2.6.dev0
